src/locale/messages.de.xlf

```diff
@@ -66,6 +66,31 @@
           <context context-type="linenumber">45</context>
         </context-group>
       </trans-unit>
+      <trans-unit id="login.login_oidc" datatype="html">
+        <source>OIDC S&apos;authentifier</source>
+        <target>OIDC Anmelden</target>
```

Comment:
The german language translation should be
"Anmelden mit OIDC",
or, probably better, simply
"Anmelden"
because an end-user need not know the login mechanism. But, if this is confusing for the use case where the user has both options, we can use
"Anmelden mit OIDC"

Reply:
Fixed, replaced with "Anmelden mit OIDC"